.github/workflows/update_manifest.yaml

```diff
@@ -0,0 +1,79 @@
+name: Update Version in Manifest
+# v99.16
+on:
+  push:
+    tags:
+      - 'v*.*'  # Matches tags in the format "vx.y"
+
+jobs:
+  update-manifest:
+    permissions:
+      contents: write
+      pull-requests: write
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v4
+
+    - name: Get version from tag
+      id: get_version
+      run: |
+        # Extracts the version number from the tag (e.g., "v1.2" -> "1.2")
+        VERSION="${GITHUB_REF##*/}"
+        VERSION="${VERSION#v}"
+        BRANCH="update-manifest-${VERSION//./-}"
+        echo "$VERSION"
+        echo "$BRANCH"
+        echo "VERSION=$VERSION" >> $GITHUB_ENV
+        echo "BRANCH=$BRANCH" >> $GITHUB_ENV
+
+    - name: Create and switch to new branch
+      run: |
+        git checkout -b "$BRANCH"
+
+    - name: Update version in manifest.json
+      run: |
+        # Read manifest.json, update the version, and overwrite the file
+        jq ".version = \"$VERSION\"" custom_components/ithodaalderop/manifest.json > custom_components/ithodaalderop/manifest_tmp.json
```

Comment:
That feels a bit ugly. What about this? 

 sed -i "s/\"version\": \"[0-9]\+\.[0-9]\+\"/\"version\": \"$VERSION\"/g" custom_components/ithodaalderop/manifest.json